governance/polls/templates/Add $ilk to Liquidations 2.0 Framework - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Add $ilk to Liquidations 2.0 Framework - $date_MONTH_DD,_YYYY
+summary: This poll proposes system parameters which could be used to initialize $ilk with the Liquidations 2.0 Framework.
+discussion_link: $risk_link
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Add $ilk to Liquidations 2.0 Framework - $date_MONTH_DD,_YYYY
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the MakerDAO mandated actors. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should abstain.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding $ilk to the Liquidations 2.0 Framework in the Maker Protocol with the parameters below:
+
+* Underlying Collateral: $collateral_asset
+* [Price Change Multiplier]($cut_link): $price_change_multiplier
```

Comment:
Ahh, this is assuming we're using exponential price function? We should maybe separate this into two sections. 
Price Function with function parameters, then the more general parameters.